lib/schemes/flat.js

```diff
@@ -6,5 +6,15 @@ module.exports = {
         var naming = bemNaming(options.naming);
```

Comment:
Well, I think this is not optimal to create the same thing many times per build.